### PR TITLE
Change M11/M12 from int16 to float32 dtype when outputting to NetCDF file

### PR DESCRIPTION
**Pull request by mliukis**
Removed compression attributes and changed M11 and M12's data types from int16 to float32

### CODE REVIEW

File: netcdf_output.py [hunk 1]
```diff
@@ -1105,25 +1105,14 @@ def netCDF_packaging(VX, VY, DX, DY, INTERPMASK, CHIPSIZEX, CHIPSIZEY, SSM, SSM1
...
-        y2 = 50
-
-        C = [(y2-y1)/(x2-x1), y1-x1*(y2-y1)/(x2-x1)]
-        # M11 = C[0]*M11+C[1]
-        var.setncattr('scale_factor', np.float32(1/C[0]))
-        var.setncattr('add_offset', np.float32(-C[1]/C[0]))
-
-        M11[noDataMask] = NoDataValue * np.float32(1/C[0]) + np.float32(-C[1]/C[0])
-        # M11[noDataMask] = NoDataValue
+        M11[noDataMask] = NoDataValue
```
Comment (jhkennedy) [inline netcdf_output.py:1108]: @mliukis for float32, wouldn't it make more sense to use `np.nan` for the NodataValue?

In the rest of this script, we use either `None` or `np.nan` for float32/64 variables:
image

Comment (mliukis) [reply netcdf_output.py:1108]: @jhkennedy hmm... We use _FillValue: -32767 for floating point values in datacubes, composites. May be a question for @alex-s-gardner?

Comment (mliukis) [reply netcdf_output.py:1108]: @jhkennedy Actually, when we say **fill_value = None**, it means that **_FillValue** is not applicable, like for **x** and **y**

Comment (jhkennedy) [reply netcdf_output.py:1108]: yes, None doesn't really matter here as it's saying there isn't missing/no data; just showing what all we do for floats. 

> Whatever NoDataValue we set, it gets converted to np.nan on read

Right, which is why I'm suggesting using np.nan.

**Review by jhkennedy — APPROVED** [review]

**Pull request merged**